tests/lib/getValidTimesFromSchedule.test.ts

```
@@ -212,4 +220,63 @@ describe('getValidTimesFromSchedule', () => {
+
+    const result = await getValidTimesFromSchedule([timeToCheck], mockEvent, []);
+
+    expect(result).toHaveLength(0); // Should be empty due to slot reservation
+  });
+
+  it('should include slots when reservations have expired', async () => {
+    // Time to check - 10:00 Monday
+    const timeToCheck = setHours(setMinutes(mockDate, 0), 10);
+    
+    // Mock expired slot reservation for this time
+    const mockExpiredReservation = {
```

ESLint

[error] 'mockExpiredReservation' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)